tests/cli/test_push.py

```diff
@@ -0,0 +1,107 @@
+    postgresql,
+    rc,
+    cli: SpintaCliRunner,
+    responses,
+    tmp_path,
+    geodb,
+    request
+):
+    create_tabular_manifest(tmp_path / 'manifest.csv', striptable('''
+    d | r | b | m | property| type   | ref     | source       | access
+    datasets/gov/example    |        |         |              |
+      | data                | sql    |         |              |
+      |   |                 |        |         |              |
+      |   |   | Country     |        | code    | salis        |
+      |   |   |   | code    | string |         | kodas        | open
+      |   |   |   | name    | string |         | pavadinimas  | open
+    '''))
+
+    # Configure local server with SQL backend
+    localrc = create_rc(rc, tmp_path, geodb)
+
+    # Configure remote server
+    remote = configure_remote_server(cli, localrc, rc, tmp_path, responses)
+    request.addfinalizer(remote.app.context.wipe_all)
+
+    # Push data from local to remote.
+    assert remote.url == 'https://example.com/'
+    result = cli.invoke(localrc, [
+        'push',
+        '-d', 'datasets/gov/example',
+        '-o', remote.url,
+        '--credentials', remote.credsfile,
+    ])
+    assert result.exit_code == 0
+    assert "Count rows:   0%" in result.stderr
+    assert "PUSH:   0%|          | 0/3" in result.stderr
+    assert "PUSH:  33%|###3      | 1/3" in result.stderr
+    assert "PUSH: 100%|##########| 3/3" in result.stderr
+
+
+def test_push_without_progress_bar(
+    postgresql,
+    rc,
+    cli: SpintaCliRunner,
+    responses,
+    tmp_path,
+    geodb,
+    request
+):
+    create_tabular_manifest(tmp_path / 'manifest.csv', striptable('''
+    d | r | b | m | property| type   | ref     | source       | access
+    datasets/gov/example    |        |         |              |
+      | data                | sql    |         |              |
+      |   |                 |        |         |              |
+      |   |   | Country     |        | code    | salis        |
+      |   |   |   | code    | string |         | kodas        | open
+      |   |   |   | name    | string |         | pavadinimas  | open
+    '''))
+
+    # Configure local server with SQL backend
+    localrc = create_rc(rc, tmp_path, geodb)
+
+    # Configure remote server
+    remote = configure_remote_server(cli, localrc, rc, tmp_path, responses)
+    request.addfinalizer(remote.app.context.wipe_all)
+
+    # Push data from local to remote.
+    assert remote.url == 'https://example.com/'
+    result = cli.invoke(localrc, [
+        'push',
+        '-d', 'datasets/gov/example',
+        '-o', remote.url,
+        '--credentials', remote.credsfile,
+        '--no-progress-bar',
+    ])
+    assert result.exit_code == 0
+    assert "Count rows:   0%" not in result.stderr
+    assert "PUSH:   0%|          | 0/3" not in result.stderr
+    assert "PUSH:  33%|###3      | 1/3" not in result.stderr
+    assert "PUSH: 100%|##########| 3/3" not in result.stderr
```

Comment:
That is not a good test, because progress bar is still shown, only in a slightly different form. Please add a better test and fix the code, to truly disable progress bar.